practice-areas/engineering/drupal/drupal-contrib-first-module-development.md

```diff
@@ -0,0 +1,15 @@
+## Drupal Contrib First module developement
+
+When a new module is needed we try to follow [Contrib First](../../../common-practices-tools/contribution/contrib-first.md), the process looks like this:
+
+1. Check with project leadership to make sure the contract allows for it.
+2. Gather requirements and identify MVP vs nice-to-haves
+3. Search for existing modules that might solve the problem. (It might be easier to stretch an existing module than build a new one)
+4. If opting to build a new module:
+    - Choose a meaningful search engine friendly module name. (crowd sourcing name suggestions is recommended)
```

Comment:
Might need to ignore the indentation problem. I am going to look for an example of indented lists.

Reply:
Here is one https://github.com/CivicActions/guidebook/blob/master/practice-areas/project-management/project-management-checklists.md?plain=1#L41

Reply:
Fixed.